packages/pv-create-component/lib/templates/galenTestTemplate.js

```diff
@@ -0,0 +1,21 @@
+module.exports = function({name, componentName, constructorName}) {
+
+  return (
+`// http://galenframework.com/docs/reference-galen-javascript-api/
+// http://galenframework.com/docs/reference-galenpages-javascript-api/
+load("../../../../js/galen/init.js");
```

Comment:
this should be customizable

Reply:
the whole Template file can be overwritten in a project if needed.